pennylane/tape/jacobian_tape.py

```diff
@@ -687,12 +687,28 @@ def hessian(self, device, params=None, **options):
 
         # The parameter-shift Hessian implementation currently only supports
         # the two-term parameter-shift rule. Raise an error for unsupported operations.
-        unsupported_ops = ("CRX", "CRY", "CRZ", "CRot")
+        supported_ops = (
+            "RX",
+            "RY",
+            "RZ",
+            "Rot",
+            "PhaseShift",
+            "ControlledPhaseShift",
+            "MultiRZ",
+            "PauliRot",
+            "U1",
+            "U2",
+            "U3",
+            "SingleExcitationMinus",
```

Comment:
SingleExcitation & DoubleExcitation are okay not to be listed here due to decompositions, correct?